packages/utils/src/namespaces.ts

```diff
@@ -228,3 +228,19 @@ export function normalizeNamespaces(
   }
   return normalizedNamespaces;
 }
+
+export function buildNamespacesFromAuth(methods: string[], accounts: string[]) {
+  accounts = accounts.map((account) => account.replace("did:pkh:", ""));
+
+  const namespace = accounts[0].split(":")[0];
```

Comment:
Aren't we assuming that all accounts are on the same namespace ?
What about wallets like Phantom that could have accounts on both `Solana` and `EVM` chains
Also could we use utils here ?
```suggestion
  const namespaces = getNamespacesFromAccounts(accounts);
```

Reply:
this first iteration supports only EVM. Later on, potentially more will be added but good suggestion. Will add it